security/compliance_operator/compliance-operator-raw-results.adoc

```diff
@@ -0,0 +1,82 @@
+[id="compliance-operator-raw-results"]
+= Retrieving Compliance Operator raw results
+include::modules/common-attributes.adoc[]
+:context: compliance-raw-results
+
+When proving compliance for your {product-title} cluster, you will most likely need to provide the scan results for auditing.
+
+.Procedure
+
+. The Compliance Operator generates and stores the raw results in a Persistent Volume. These results are in Asset Reporting Format (ARF). To view these results, you can explore the aforementioned ComplianceSuite object:
++
+[source,terminal]
+----
+$ oc get compliancesuites nist-moderate -o json \
+| jq '.status.scanStatuses[].resultsStorage'
+{
+  "name": "rhcos4-moderate-worker",
+  "namespace": "openshift-compliance"
+}
+{
+  "name": "rhcos4-moderate-master",
+  "namespace": "openshift-compliance"
+}
+----
++
+This shows the Persistent Volume Claims where the raw results are accessible.
+
+. You can verify this by using the name and namespace of one of the results:
++
+[source,terminal]
+----
+$ oc get pvc -n openshift-compliance rhcos4-moderate-worker
+----
++
+.Example output
+[source,terminal]
+----
+NAME                 	STATUS   VOLUME                                 	CAPACITY   ACCESS MODES   STORAGECLASS   AGE
+rhcos4-moderate-worker   Bound	pvc-548f6cfe-164b-42fe-ba13-a07cfbc77f3a   1Gi    	RWO        	gp2        	92m
+----
+
+. Fetching the raw results is possible by spawning a pod that mounts the volume, and copying the results:
++
+.Example pod
+[source,yaml]
+----
+apiVersion: "v1"
+kind: Pod
+metadata:
+  name: pv-extract
+spec:
+  containers:
+    - name: pv-extract-pod
+        image: registry.access.redhat.com/ubi8/ubi
+        command: ["sleep", "3000"]
+        volumeMounts:
```

Comment:
`name`, image`, `command` and `volumeMounts` should go in the same identation level. `volumeMounts` then should contain the path list.